google-cloud/src/pubsub/topic.rs

```diff
@@ -40,23 +40,27 @@ impl Topic {
         }
     }
 
+    /// Returns the unique identifier within its project
+    pub fn id(&self) -> &str {
+        self.name
+            .rfind('/')
+            .map(|i| &self.name[i + 1..])
+            .expect("bad topic name")
```

Comment:
Same suggestion as the other occurence, in `Subscription`
```suggestion
        self.name
            .split("/")
            .last()
            .expect("empty subscription name")
```